.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Have you checked the preview of this?
I think you are missing some line breaks to get a good formatting.

Reply:
Can you exactly point where?! - that would be helpful.

Reply:
I just noticed that you are not using headers at all. I think it could be more organized if you put things like "Describe the bug", "To reproduce" and so on in headers.

Reply:
It's a generic template that we get from GitHub. For readability, I agree. 
This has been modified.